mkdocs/tests/build_tests.py

```diff
@@ -11,12 +11,13 @@
 from mkdocs import build, nav, config
 from mkdocs.exceptions import MarkdownNotFound
 from mkdocs.tests.base import dedent
+from mkdocs.build import convert_markdown
```

Comment:
Hmm, that sounds weird to me. If we are getting this many circular imports we are doing something wrong.